### PR TITLE
redirector play

### PR DESCRIPTION
**Pull request by fredsmith**

### CODE REVIEW

File: playbooks/edx-east/redirector.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      NGINX_ENABLE_SSL: "{{ REDIRECT_NGINX_ENABLE_SSL }}"
+      NGINX_SSL_CERTIFICATE: "{{ REDIRECT_NGINX_SSL_CERTIFICATE }}"
+      NGINX_SSL_KEY: "{{ REDIRECT_NGINX_SSL_KEY }}"
+    - role: datadog
+      when: COMMON_ENABLE_DATADOG
+    - role: splunkforwarder
+      when: COMMON_ENABLE_SPLUNKFORWARDER
+    - role: newrelic
+      when: COMMON_ENABLE_NEWRELIC
+    - role: minos
```
Comment (jibsheet) [inline playbooks/edx-east/redirector.yml]: I don't think we want minos on for this play (even optionally)

**Comment by jibsheet** [comment]
See my notes in https://github.com/edx-ops/edx-secure/pull/902/files about structuring the data we pass around.  I'm worried about the refactor we'll need to do when we need a second redirect on the same host.

File: playbooks/roles/nginx/templates/edx/app/nginx/sites-available/nginx_redirect.j2 [hunk 2]
```diff
@@ -5,10 +5,10 @@
 {%- endif -%}
 
 server {
-  listen {{ EDXAPP_LMS_NGINX_PORT }} {{ default_site }};
+  listen {{ REDIRECT_NGINX_PORT }} {{ default_site }};
```
Comment (e0d) [inline playbooks/roles/nginx/templates/edx/app/nginx/sites-available/nginx_redirect.j2:8]: Are you confident we're not depending on this anywhere?

File: playbooks/edx-east/redirector.yml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+- name: Deploy redirector host
+  hosts: all
+  sudo: True
+  gather_facts: True
+  vars:
+    serial_count: 1
```
Comment (maxrothman) [inline playbooks/edx-east/redirector.yml:6]: No more freakin serial_counts!

File: playbooks/edx-east/redirector.yml [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  serial: "{{ serial_count }}"
+  roles:
+    - aws
+    - role: nginx
+      nginx_redirects: "{{ NGINX_REDIRECTOR_CUSTOM_REDIRECTS }}"
+      REDIRECT_NGINX_PORT: "80"
+      REDIRECT_SSL_NGINX_PORT: "443"
+      NGINX_ENABLE_SSL: "{{ REDIRECT_NGINX_ENABLE_SSL }}"
+      NGINX_SSL_CERTIFICATE: "{{ REDIRECT_NGINX_SSL_CERTIFICATE }}"
+      NGINX_SSL_KEY: "{{ REDIRECT_NGINX_SSL_KEY }}"
```
Comment (maxrothman) [inline playbooks/edx-east/redirector.yml:17]: Where are all these vars defined?

**Pull request merged**